streaming/base/format/mds/writer.py

```diff
@@ -45,7 +45,7 @@ def __init__(self,
         self.column_sizes = []
         for name in sorted(columns):
             encoding = columns[name]
-            assert is_mds_encoding(encoding)
+            assert is_mds_encoding(encoding), f'MDSWriter passed column "{name}" with encoding "{encoding}" that is not supported. Supported encodings are {get_mds_encodings()}'
```

Comment:
Thank you for making this change. One minor suggestion, can you please replace `assert` with `TypeError` Exception ? We are planning to revisit the code and replace all the `assert` statement with appropriate Exception type with meaningful message.
```suggestion
            if not is_mds_encoding(encoding):
                raise TypeError(f'MDSWriter passed column "{name}" with encoding "{encoding}" is unsupported. Supported encodings are {get_mds_encodings()}')
```